### PR TITLE
Free disk space

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: .github/workflows/build-test-cxx.yml [hunk 1]
```diff
@@ -54,6 +54,11 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
+    - name: Free Disk Space (Ubuntu)
+      uses: jlumbroso/free-disk-space@main
```
Comment (tbirdso) [inline .github/workflows/build-test-cxx.yml:58]: For safety reasons let's use a tagged version.
```suggestion
      uses: jlumbroso/free-disk-space@v1.3.1
```

File: .github/workflows/build-test-package-python.yml [hunk 2]
```diff
@@ -62,14 +62,10 @@ jobs:
...
-    - name: 'Free up disk space'
-      run: |
-        # Workaround for https://github.com/actions/virtual-environments/issues/709
-        df -h
-        sudo apt-get clean
-        sudo rm -rf "/usr/local/share/boost"
-        sudo rm -rf "$AGENT_TOOLSDIRECTORY"
-        df -h
+    - name: Free Disk Space (Ubuntu)
+      uses: jlumbroso/free-disk-space@main
```
Comment (tbirdso) [inline .github/workflows/build-test-package-python.yml:66]:
```suggestion
      uses: jlumbroso/free-disk-space@v1.3.1
```

File: .github/workflows/build-test-package-python.yml [hunk 3]
```diff
@@ -140,6 +136,11 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
+    - name: Free Disk Space (Ubuntu)
+      uses: jlumbroso/free-disk-space@main
```
Comment (tbirdso) [inline .github/workflows/build-test-package-python.yml:140]:
```suggestion
      uses: jlumbroso/free-disk-space@v1.3.1
```

File: .github/workflows/build-test-package-python.yml [hunk 4]
```diff
@@ -214,6 +215,11 @@ jobs:
       with:
         path: "im"
 
+    - name: Free Disk Space (Ubuntu)
+      uses: jlumbroso/free-disk-space@main
```
Comment (tbirdso) [inline .github/workflows/build-test-package-python.yml:219]:
```suggestion
      uses: jlumbroso/free-disk-space@v1.3.1
```

**Review by tbirdso — CHANGES_REQUESTED** [review]

**Comment by tbirdso** [comment]
xref https://github.com/InsightSoftwareConsortium/ITKSplitComponents/pull/74

**Comment by thewtex** [comment]
Also needs updates in #69

**Comment by tbirdso** [comment]
Closing in favor of https://github.com/InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/pull/69 .

**Pull request closed without merging**